tests/py2puml/inspection/test_inspectclass.py

```diff
@@ -148,3 +149,33 @@ def test_inspect_module_should_unwrap_decorated_constructor():
     # PointDecoratedWithoutWrapping UmlClass
     point_without_wrapping_umlitem: UmlClass = domain_items_by_fqn['tests.modules.withwrappedconstructor.PointDecoratedWithoutWrapping']
     assert len(point_without_wrapping_umlitem.attributes) == 0, 'the attributes of the original constructor could not be found, the constructor was not wrapped by the decorator'
+
+
+def test_inspect_module_should_find_methods():
+    """ Test that methods are detected including static methods
+
+    This test case assumes that the methods will be sorted by type as follow:
+        1a - instance methods (special methods aka "dunder")
```

Comment:
instance methods usually are not dunder methods. Point.get_coordinates() is an instance method, for example